boards/tsi/ek_tsi_skyp/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2018-2021 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_EK_TSI_SKYP
+
+# MPU-based null-pointer dereferencing detection cannot
+# be applied as the (0x0 - 0x400) is unmapped but QEMU
+# will still permit bus access.
+choice NULL_POINTER_EXCEPTION_DETECTION
+	bool
+	default NULL_POINTER_EXCEPTION_DETECTION_NONE if QEMU_TARGET
+endchoice
+
+if SERIAL
+
+config UART_INTERRUPT_DRIVEN
+	default n
```

Comment:
Hi Ganesh, do you need to disable this UART_INTERRUPT_DRIVEN? Is this because of jtag driver is in polling mode only ?

Reply:
Yes, we are using the polling method. That's why I disabled the feature.